scripts/linux/createArmBase.sh

```diff
@@ -142,7 +142,7 @@ process_args()
         print_help_and_exit
     fi
 
-    registries=("edgebuilds.azurecr.io/" "edgerelease.azurecr.io/" "")
+    registries=("")
```

Comment:
screen this script file to update comments, e.g. line7-8.

Reply:
Updated.